example/index.ts

```diff
@@ -1,55 +1,37 @@
-import { rAF, forceV, applyForce, vector as Vector, Entity } from "../src";
-
-interface UseGravityArgs {
-  moverMass: number;
-  attractorMass: number;
-  attractor: Vector<number>;
-}
-
-const applyForceForStep = (config: UseGravityArgs, mover: Entity) => {
-  const force = forceV({
-    ...config,
-    mover: mover.position
-  });
-
-  return applyForce({
-    force,
-    entity: {
-      mass: config.moverMass,
-      acceleration: mover.acceleration,
-      velocity: mover.velocity,
-      position: mover.position
+import { gravity, vector } from "../src";
+
+const el = document.createElement("div");
+document.body.appendChild(el);
+
+const positionAttractor = [500, 500] as vector<number>;
+const attractorEl = document.createElement("div");
+document.body.appendChild(attractorEl);
+
+el.style.height = "10px";
+el.style.width = "10px";
+el.style.borderRadius = "50%";
+el.style.background = "steelblue";
```

Comment:
great color 👏

Reply:
Stolen from all the `d3` tutorials I did back in the day 😂

Reply:
🔥